src/general/NewGameSettings.cs

```diff
@@ -301,6 +311,10 @@ public override void _Ready()
             difficultyPresetAdvancedButton.AddItem(preset.UntranslatedName);
         }
 
+        patchMapExplorationMode.AddItem(TranslationServer.Translate("FOG_OF_WAR_DISABLED"));
```

Comment:
Could you add a comment to the enum for the exploration mode that the items may not be reordered, or their values may not change as otherwise the new game GUI will break?